front-end/src/pages/ManageProjects.js

```diff
@@ -0,0 +1,22 @@
+// This is the Manage Project page.
+
+import React from 'react';
+import AddProject from "../components/AddProject";
+import ProjectsList from "../components/ProjectsList";
+import Navbar from "../components/Navbar";
+
+
+export default function ManageProjects() {
+    return (
+      <div>
+          <Navbar />
+          <br />
+
+          <h3> Add a new project: </h3>
+          <AddProject />
+          <br />
+          <h3> Select an existing project: </h3>
+          <ProjectsList />
+      </div>
```

Comment:
I really like that you created different small components nested within each component to reduce the complexity of each file, it makes it easier to read and understand code.